src/components/BigNumberPad.js

```diff
@@ -17,27 +18,64 @@ const padNumbers = [
     ['.', '0', '<'],
 ];
 
-const BigNumberPad = props => (
-    <View style={[styles.flexColumn, styles.w100]}>
-        {_.map(padNumbers, (row, rowIndex) => (
-            <View key={`NumberPadRow-${rowIndex}`} style={[styles.flexRow, styles.mt3]}>
-                {_.map(row, (column, columnIndex) => {
-                    // Adding margin between buttons except first column to
-                    // avoid unccessary space before the first column.
-                    const marginLeft = columnIndex > 0 ? styles.ml3 : {};
-                    return (
-                        <ExpensifyButton
-                            key={column}
-                            style={[styles.flex1, marginLeft]}
-                            text={column}
-                            onPress={() => props.numberPressed(column)}
-                        />
-                    );
-                })}
+class BigNumberPad extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {timer: null};
+    }
+
+    /**
+     * Handle long press key on number pad.
+     * Only handles the '<' key and starts the continuous input timer.
+     *
+     * @param {String} key
+     */
+    handleLongPress(key) {
+        // Only handles deleting.
+        if (key !== '<') {
+            return;
+        }
```

Comment:
This should be part of BigNumberPad. This should be handled in the parent component. The only job of this method should be to call the long press callback on time intervals emitting continuous long-press.

Reply:
Sorry, but I don't quite understand. There is no more long press callback in that context. I think we have agreed that this entire logic should be moved to BigNumberPad component.

I have just reread the thread above and @iwiznia clearly says "shouldn't all this logic be on the BigNumberPad component". If we are to do
```js
        if (key !== '<') {
            return;
        }
```
inside parent component we would need to also pass some new arg to callback to indicate that the keyPress was started by the timer.  And overall moving it to parent component would mean that we'll have timers on every button and not just the `<`.

Reply:
I agree with @dklymenk, not sure why would anyone want to handle the long presses differently. But even if at some point we do want that, we can extract the logic then.

Reply:
OK, I give it a go for now. I still feel that all keys should have the same behaviour irrespective of the Key. If we only want to handle long-press for `<` then it should be a parent to control that.